_tests/unit_tests/fixtures/forms.html

```diff
@@ -0,0 +1,135 @@
+<form id="email-subscribe-form" action="/subscriptions/new/" method="POST" enctype="application/x-www-form-urlencoded">
+
+    <div class="form-group">
+        <label for="email-subscribe-form_email-1">
+            Email address
+        </label>
+        <input id="email-subscribe-form_email-1" type="email" name="email1" placeholder="example@mail.com" required="">
+    </div>
+    <p class="short-desc u-mt10">
+        The information you provide will permit the Consumer Financial
+        Protection Bureau to process your request or inquiry.
+        <a href="/privacy-policy/">See More</a>
+    </p>
+    <div class="form-group">
+        <input type="hidden" name="code" value="USCFPB_91">
+        <input class="btn" type="submit" value="Sign up">
+    </div>
+
+</form>
+
+<form class="js-validate-filters" method="get">
+
+    <div class="form-l form-l__float form-l__flush">
+
+        <div class="form-l_col form-l_col-1-3">
+            <div class="form-group required-check-group">
+                <label class="form-label-header">
+                    Calendars
+                </label>
+                <label class="form-group_item custom-input_label is-enabled"><span class="custom-input_text">Richard Cordray
+                </span><input class="custom-input custom-input_input" type="checkbox" name="filter_calendar" value="Richard Cordray"><span class="custom-input_checkbox"></span></label>
+                <label class="form-group_item custom-input_label is-enabled"><span class="custom-input_text">Steve Antonakes
+                </span><input class="custom-input custom-input_input" type="checkbox" name="filter_calendar" value="Steve Antonakes"><span class="custom-input_checkbox"></span></label>
+                <label class="form-group_item custom-input_label is-enabled"><span class="custom-input_text">Raj Date
+                </span><input class="custom-input custom-input_input" type="checkbox" name="filter_calendar" value="Raj Date"><span class="custom-input_checkbox"></span></label>
+                <label class="form-group_item custom-input_label is-enabled"><span class="custom-input_text">Elizabeth Warren
+                </span><input class="custom-input custom-input_input" type="checkbox" name="filter_calendar" value="Elizabeth Warren"><span class="custom-input_checkbox"></span></label>
+            </div>
+        </div>
+
+        <div class="form-l_col form-l_col-2-3">
+          <div class="form-l form-l__flush">
+            <label class="form-l_col-1 form-label-header">
+                Date range
+            </label>
+
+            <div class="form-l_col form-l_col-1-2 js-filter_range-date-wrapper" style="display: none;">
```

Comment:
Refactor to remove inline style and see if a hidden class could be used instead (e.g. from CF).